packages/quantic/force-app/examples/main/lwc/actionAddCustomSortOptions/actionAddCustomSortOptions.html

```diff
@@ -0,0 +1,10 @@
+<template>
+  <lightning-button
+    label={label}
+    title={label}
+    onclick={handleAddCustomSortOptions}
+    disabled={disabled}
+    class="configurator__add-Custom-Sort-Options"
```

Comment:
Why do you need this CSS class? and why the casing is kinda random?

Reply:
thats a good question :D